EarTrumpet/UI/ViewModels/DeviceViewModel.cs

```diff
@@ -10,6 +11,16 @@ namespace EarTrumpet.UI.ViewModels
 {
     public class DeviceViewModel : AudioSessionViewModel, IDeviceViewModel
     {
+        public class DisplayNameComparer : IComparer<DeviceViewModel>
+        {
+            public int Compare(DeviceViewModel one, DeviceViewModel two)
+            {
+                return string.Compare(one.DisplayName, two.DisplayName, StringComparison.Ordinal);
```

Comment:
```suggestion
                return string.Compare(one.DisplayName, two.DisplayName, StringComparison.CurrentCultureIgnoreCase);
```

Languages have different sorting rules, so let's make sure we respect them.
https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#:~:text=Data%20that%20requires%20local%20linguistic%20customs.